Core/HLE/sceAtrac.cpp

```diff
@@ -1964,6 +1964,8 @@ static const u16 at3plusHeaderMap[][3] = {
 	{ 0x4524, 0x0, 0x0 },
 	{ 0x02E8, 0x1, 0x0 },
 	{ 0x5C24, 0x0, 0x0 },
+	{ 0x0118, 0x1, 0x0 },
+	{ 0x2224, 0x0, 0x0 },
```

Comment:
the https://github.com/hrydgard/ppsspp/pull/7530/files#diff-a4d8150342953ac0053b961c4601e20fL2013 
it's i+=2
If don't add a pair, the following thing would miss.